Description: Upstream changes introduced in version 2.0.0~alpha2+nolibs-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 librecad (2.0.0~alpha2+nolibs-1) experimental; urgency=low
 .
   * New upstream release, experimental developmental
   * Removed dependency on libqt4-sql-sqlite since help is no longer
     included
 .
 The person named in the Author field signed this changelog entry.
Author: Scott Howard <showard@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

Index: librecad/libraries/jwwlib/jwwlib.pro
===================================================================
--- librecad.orig/libraries/jwwlib/jwwlib.pro	2012-07-02 19:16:48.000000000 -0400
+++ librecad/libraries/jwwlib/jwwlib.pro	2012-07-02 19:21:11.142934950 -0400
@@ -21,8 +21,8 @@
 include(../../settings.pro)
 include(../../common.pro)
 
-#INCLUDEPATH += \
-#    ../dxflib/src
+INCLUDEPATH += \
+    /usr/include/dxflib
 
 SOURCES += \
     src/dl_writer_ascii.cpp \
